sysdeps/unix/sysv/linux/Makefile

```diff
@@ -55,6 +55,7 @@ endif
 
 ifeq ($(subdir),misc)
 sysdep_routines += \
+  uk_vsyscall \
```

Comment:
Can you please add in the commit message a brief description of the changes from this commit (e.g. why it's needed, what is changed). Thank you.

Also, an usage example / test scenario would be useful to be added in the PR description.